### ISSUE TITLE
Remove Bootstrap dependency from the front-end navbar

### ISSUE THREAD

**Issue by vnenkpet** [noumo/easyii#39]
Sometimes loading the bootstrap css and js files can break your layout if your front-end doesn't use bootstrap at all. Can we create a new Asset that only affects easyii-navbar and is loaded alone instead of the Bootstrap asset when not in the /admin area?

**Comment by noumo** [comment]
Yes this is good idea, but we need to reproduce frontend navbar on pure css, fortunately he is very simple :)

**Comment by YaroslavShapoval** [comment]
I am thinking about the separation of easyii-navbar style and bootstrap style. It is pretty simple to implement, but still have some questions:
1. Should it be responsive like bootstrap's container?
2. Should I move needed glyphicons to png-icons?
3. Why opensans googlefont registered here?

**Comment by lial** [comment]
Подскажите как отключается bootsrap для форнтенда ? Летит вся вся верстка из-за него

**Comment by sim2github** [comment]
@lial  Бандлы можно переназначить в конфиге 

``` php
    'components' => [
        'assetManager' => [
            'class' => 'yii\web\AssetManager',
            'linkAssets' => true,
            'appendTimestamp' => YII_ENV_DEV,
            'bundles' => [
                'yii\bootstrap\BootstrapAsset' => [
                    'sourcePath' => null, 'css' => [], 
                ],
                'yii\bootstrap\BootstrapPluginAsset' => [
                    'sourcePath' => null, 'js' => [],
                ],
                'yii\jui\JuiAsset' => [
                    'sourcePath' => null, 'css' => [], 'js' => [],
                ],
            ],
        ],
```

**Comment by lial** [comment]
Я спрашивал как отключать их для фронта только, то что указали вы есть в манах Yii и оно проблему не решает, ибо отключает их как для фронта так и для бэкенда, чуствуете разницу ?

**Comment by sim2github** [comment]
Тогда внимательнее прочитайте маны по [yii\web\AssetBundle](https://yiiframework.com.ua/ru/doc/guide/2/structure-assets/)

##### app\assets\AppAsset

``` php
    public $depends = [
        'yii\web\JqueryAsset',
        //'yii\bootstrap\BootstrapAsset',
    ];
```

Данный код уберет из подгрузки зависимость bootstrap.css.
Я правильно прочувствовал разницу?
